### PR TITLE
enabling ssl from the web panel

### PR DESCRIPTION
**Pull request by Mohit-3196**
https://github.com/ant-media/Ant-Media-Server/issues/5320

### CODE REVIEW

File: docs/guides/installing-on-linux/setting-up-ssl.md [hunk 1]
```diff
@@ -147,4 +147,23 @@ If the scripts above return successfully, SSL will be installed on your server,
...
+
+However, with the release of Ant Media Server version 2.6.2, we have streamlined the SSL enablement process, allowing users to seamlessly secure their media server directly from the AMS Web Panel. This empowers users to enable SSL with utmost ease and convenience.
+
+- After [installaling the Ant Media Server](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/), login to the web panel and navigate to SETTINGS > SSL.
+![](@site/static/img/ssl-webpanel/ssl-settings.png)
+
+- Under Type filed and you can choose among various options to enable SSL like [using your own domain](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/#option-2-create-lets-encrypt-certificate-with-http-01-challenge), [free subdomain of antmedia.cloud](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/#option-1-gets-a-free-subdomain-and-install-ssl-with-lets-encrypt), or to [import your own certificate](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/#option-3-import-your-custom-certificate) and then click Activate to enable the SSL and restart your server.
```
Comment (timantmedia) [inline docs/guides/installing-on-linux/setting-up-ssl.md:160]: I think you meant "field" not "filed" - "Under Type field"

**Review by timantmedia — CHANGES_REQUESTED** [review]

File: docs/guides/installing-on-linux/setting-up-ssl.md (hunk 1 above)
Comment (Mohit-3196) [reply docs/guides/installing-on-linux/setting-up-ssl.md]: Added the comment and removed the typo.

**Comment by Mohit-3196** [comment]
@timantmedia 
I had corrected the typo and revised the PR.
Can you please check and merge this if everything is OK?

Thank you

**Review by timantmedia — APPROVED** [review]

**Comment by timantmedia** [comment]
@Mohit-3196 merged, thanks.

**Comment by Mohit-3196** [comment]
Thank you @timantmedia

**Pull request merged**